wpilibj/src/main/java/edu/wpi/first/wpilibj/kinematics/DifferentialDriveKinematics.java

```diff
@@ -0,0 +1,62 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package edu.wpi.first.wpilibj.kinematics;
+
+/**
+ * Helper class that converts a chassis velocity (dx and dtheta components) to
+ * left and right wheel velocities for a differential drive.
+ *
+ * <p>Inverse kinematics converts a desired chassis speed into left and right
+ * velocity components whereas forward kinematics converts left and right
+ * component velocities into a linear and angular chassis speed.
+ */
+public class DifferentialDriveKinematics {
+  private final double m_driveRadiusMeters;
```

Comment:
See comment about naming in C++

Reply:
What else would we call it? Differential drive kinematics is exactly what this is. The classes in the drive folder are normalized versions of various inverse kinematics schemes.

Reply:
I am talking about the variable... https://github.com/wpilibsuite/allwpilib/pull/1787#discussion_r315506149

Reply:
ok. I thought you meant the class because your comment mentioning the "comment about naming" was ambiguous.

Reply:
Any suggestions for what it could be renamed to? @calcmogul